webodm.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+platform="Linux" # Assumed
+uname=`uname`
+if [[ $uname == "Darwin" ]]; then
+	platform = "MacOS / OSX"
+elif [[ $uname == MINGW* ]]; then
+	platform = "Windows"
+fi
+
+usage(){
+  echo "Usage: $0 <command> [options]"
+  echo
+  echo "This program helps to manage the setup/teardown of the docker containers for running WebODM. We recommend that you read the full documentation of docker at https://docs.docker.com if you want to customize your setup."
+  echo 
+  echo "Command list:"
+  echo "	start		Start WebODM"
+  echo "	stop		Stop WebODM"
+  echo "	update		Update WebODM to the latest release"
+  echo "	rebuild		Rebuild all docker containers and perform cleanups"
+  exit
+}
+
+check_command(){
+	check_msg="\e[92m\e[1m OK\e[0m\e[39m"
+	hash $1 2>/dev/null || not_found=true 
+	if [[ $not_found ]]; then
+		check_msg="\e[91m can't find $1! Check that the program is installed before launching WebODM. $2\e[39m"
+	fi
+
+	echo -e "Checking for $1... $check_msg"
+	
+	if [[ $not_found ]]; then
+		exit 1
```

Comment:
Using `set -e` (above) means that this can `return 1` instead and be useful as a test, rather than something that always terminates.